API/api/services/event_service.py

```
@@ -1,13 +1,43 @@
+                "detail": "Modification impossible: des inscriptions existent déjà."
+            })
+
+        event.name = event_data["name"]
+        event.event_date = event_date
+
+        try:
+            return self.repo.update_event(event)
+        except IntegrityError:
+            raise ValidationError({
+                "event_date": "Un événement existe déjà pour cette date."
+            })
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)